### PR TITLE
Created MPLAB Project and I2C Driver

### PR DESCRIPTION
**Pull request by LanternH**

### CODE REVIEW

File: lcd_ssd1306.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    data[6] = SSD1306_COM_PINS_HARDWARE_CONFIG;         /* Set COM Pins hardware configuration */
+    data[7] = SSD1306_SEQ_COM_PIN_CONFIGURATION;        /* Sequential COM pin configuration */
+    data[8] = SSD1306_SET_CONTRAST;                     /* Set Contrast Control */
+    data[8] = SSD1306_INIT_CONTRAST_VALUE;              /* Set initial Contrast value */                               
+    data[9] = SSD1306_ENTIRE_DISPLAY_ON;                /* Disable Entire Display On */
+    data[10] = SSD1306_SET_NORMAL_DISPLAY;              /* Set Normal Display */
+    data[11] = SSD1306_SET_DISPLAY_CLOCK;               /* Set Osc Frequency */
+    data[12] = SSD1306_DISPLAY_CLK_DIVIDE_RATIO;        /* Divide ratio = 1. Oscilator freq = 8 */
+    data[13] = SSD1306_CHARGE_PUMP_SETTING;             /* Enable charge pump regulator */
+    data[14] = SSD1306_CHARGE_PUMP_VALUE;               /* Display On */
```
Comment (LanternH) [inline lcd_ssd1306.c]: Check the numbers of the array

File: lcd_ssd1306.h [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+#define SSD1306_SET_DISPLAY_OFF             0xAE /* Display ON in normal mode */
+
+/* SCROLLING COMMAND TABLE*/
+#define SSD1306_RIGHT_SCROLL                0x26 /* Right Horizontal Scroll 1 column */
+#define SSD1306_LEFT_SCROLL                 0x27 /* Left Horizontal Scroll 1 column */
+#define SSD1306_VERTICAL_RIGHT_SCROLL       0x29 /* Vertical and right scroll (Horizontal scroll by 1 column) */
+#define SSD1306_VERTICAL_LEFT_SCROLL        0x2A /* Vertical and left scroll (Horizontal scroll by 1 column) */
+#define SSD1306_DUMMY_BYTE                  0x00 /* Dummy Byte */
+#define SSD1306_START_PAGE                  0x00 /* Initialize start page on page zero */
+#define SSD1306_SCROLL_STEP                 0x02 /* 010b ? 128 fvrames between each scroll */
```
Comment (LanternH) [inline lcd_ssd1306.h]: fvrames typo error 'frames'

File: lcd_ssd1306.h [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+#define SSD1306_PAGE_START                  0x00 /* Page start Address, range : 0-7d, (RESET = 0d). This command is only for horizontal or vertical addressing mode */
+#define SSD1306_PAGE_END                    0x07 /* Page end Address, range : 0-7d, (RESET = 7d). This command is only for horizontal or vertical addressing mode */
+#define SSD1306_SET_PAGE_START_ADDRESS      0xB0 /* Setup page start and end address B0~B7 (PAGE0~PAGE7) This command is only for page addressing mode */
+
+/* HARDWARE CONFIGURATION (PANEL RESOLUTION & LAYOUT RELATED) COMMAND TABLE */
+#define SSD1306_SET_DISPLAY_START_LINE      0x40 /* Set display RAM display start line register from 0-63. 0x40 during RESET */
+#define SSD1306_SEGREMAP                    0xA0 /* 0xA0 column address 0 is mapped to SEG0 (RESET). 0xA1 column address 127 is mapped to SEG0 */
+#define SSD1306_SETMULTIPLEX                0xA8 /* See DataSheet RESET= 111111b */
+#define SSD1306_SETMULTIPLEX_VALUE          0x3F /* See DataSheet RESET= 111111b */
+#define SSD1306_COM_OUTPUT_SCAN_DIRECTION   0xC0 /* normal mode (RESET) Scan fromCOM0 to COM[N ?1]. remapped mode. Scan from COM[N-1] to COM0 Where N is the Multiplex ratio */
```
Comment (LanternH) [inline lcd_ssd1306.h]: add a space between 'fromdCOM0'

**Review by Tonyxf2 — APPROVED** [review]
Looks great

**Pull request merged**